subfun/BIDS_RFX.m

```diff
@@ -76,17 +76,17 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
                 % FFX Directory
                 ffxDir = getFFXdir(subNumber, mmFunctionalSmoothing, opt);
                 conImg = spm_select('FPlist', ffxDir, '^con*.*nii$');
-                matlabbatch{1}.spm.spatial.smooth.data = cellstr(conImg);
```

Comment:
Initially matlabbatch{1} was overwritten with every new subject. Therefore only the last subject was processed, and the rest were overwritten.